### DISCUSSION TITLE
why there is only one result for one tissue-specific-GEM?

### DISCUSSION THREAD

**Discussion by Joyjoyjoyc** [SysBioChalmers/Human-GEM#831] [Q&A]
Hi, everyone!
I followed the tutorial of Human-GEM and use my own bulk RNA-seq data to extract the personalized GEM, but I wonder how to build a tissue-specific GEM model with only one result? which I mean as I can see that in the example file 'tINIT_GTEx_outputs.mat', there are 30 tissue-specific-GEM, but does one tissue should be compound of many samples, so why does it only contain only one GEM for one tissue? or maybe they only choose one sample's expression data as a representative of the tissue? so I'd like to know how to exactly generate a tissue-specifc GEM and compare with others?  
Thanks!

**Comment by JonathanRob** [comment]
Hi @Joyjoyjoyc, the original implementation of tINIT ([Agren et al. 2014](https://www.embopress.org/doi/full/10.1002/msb.145122)) calculated gene scores for a tissue by **comparing gene expression values in that tissue to the mean expression of those genes across several tissues**, and therefore required gene expression data for multiple tissues to be provided.

However, the updated version of tINIT ([Robinson et al. 2020](https://www.science.org/doi/10.1126/scisignal.aaz1482)) allowed the specification of an **expression threshold**, to which gene expression values could be compared instead of to average values across several tissues. This therefore allows you to create a tissue-specific GEM **without requiring expression data for multiple tissues** - you only need the expression data for the tissue(s) for which you wish to generate a context-specific GEM.

The most recent version of the algorithm is ftINIT ([Gustafsson et al. 2023](https://www.pnas.org/doi/10.1073/pnas.2217868120)), which improves, among many other things, the **speed and robustness of the model generation process**. This is the version that we currently recommend, and the user guide can be found [here](https://sysbiochalmers.github.io/Human-GEM-guide/gem_extraction/).

**Reply by Joyjoyjoyc** [reply]
Hi @JonathanRob, thanks for your reply! I was indeed using the ftINIT to extract my tissue-specific GEM, and it is exactly fast and convenient. 
But I still want to know how to compare 2 GEMs between different tissues like the figure3 in the paper _An atlas of human metabolism_ given below. My real question is how to pick the representative sample for one tissue, and is it possible to use all samples in a tissue to make a comparison? 
Because there will be many samples(and also many expression data) in one tissue, and one expression data is corresponding to one GEM, so did you only choose one sample as a representation but not all of the samples in this tissue to make a comparison with other tissues? if so, how to pick the representative sample for one tissue because of the heterogeneity?
Thanks!
![image](https://github.com/SysBioChalmers/Human-GEM/assets/157622479/0152d531-86a7-4514-9870-02343dd5e3b4)

**Comment by JonathanRob** [comment]
Ah my mistake, I misunderstood different "samples" in your original question to mean different "tissues".

Yes, for our analysis we simply took the average expression value (TPM) of all samples for a tissue to use as input for generating its tissue-specific GEM. This of course does not capture the expression heterogeneity within the tissue, since it will be an average profile.

One could instead generate a sample-specific GEM for every sample, which would give a bit more insight into how the within-tissue sample expression differences impact the resulting GEMs. This was something we did in the [ftINIT paper](https://www.pnas.org/doi/10.1073/pnas.2217868120) (see e.g. Fig. 2), where we looked at structural and functional variation among GEMs generated from different samples of the same tissue (or different groups ["pools"] of single cells from the same cell type).

**Reply by Joyjoyjoyc** [reply]
Thank you so much for your reply!!! I really understand this time. I did make a structure comparison same as the [ftINIT paper](https://www.pnas.org/doi/10.1073/pnas.2217868120)(Fig.2D) in my bulk RNA-seq, and maybe I can follow the scRNA-seq instruction to make more interesting comparisons in my data.
Have a nice day!
